test/functional/features/overview-page.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Overview page
+  In order complete my appeal
+  As a citizen
+  I want to know what stage of the appeal I am at
+
+  Scenario: Entering my contact details should open next task
+    Given I am on the "overview" page
+    When I click "Continue" button
+    Then I expect to be redirect back to the task-list
```

Comment:
Tests are failing, please review the pipeline and make sure all the tests are passing